### PR TITLE
Home screen (1/2)

### PR DESCRIPTION
**Pull request by lbkchen**
* Can scroll through veterans
* Can connect through button on card
* Can view profile by pressing card
* Can connect through profile view
* Added underscore library 

home-screen

### CODE REVIEW

**Comment by lbkchen** [comment]
I swear the circles are not completely round...

File: src/screens/HomeScreen.js [hunk 1]
```diff
@@ -1,17 +1,129 @@
...
+
+  navigateToProfile(params) {
+    this.props.navigation.navigate('HomeProfile', params);
+  }
+
+  /**
+   * TODO (Claire): You can return all your stuff here
+   */
+  renderResources() {
+    return null;
```
Comment (lbkchen) [inline src/screens/HomeScreen.js:77]: @clairelin135 you can just dump your stuff here

**Comment by lbkchen** [comment]
#21

File: src/components/ProfileCard.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+export default class ProfileCard extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.connectWithVeteran = this.connectWithVeteran.bind(this);
+  }
+
+  connectWithVeteran(event, onSuccess, onFailure) {
```
Comment (nikitr) [inline src/components/ProfileCard.js:39]: since we are using the same logic in the ConnectBox, we can pull this out later to a common functions file

**Review by nikitr — APPROVED** [review]
LGTM

**Pull request merged**